build.gradle.kts

```diff
@@ -165,6 +170,7 @@ allprojects {
 				excludeGroup("ca.weblite")
 			}
 		}
+		mavenLocal()
```

Comment:
This should probably be removed before merging.

Reply:
is it bad in general to have it here? I feel like it would be fine to keep

Reply:
I'd prefer it not be in master. You can keep it locally just don't commit it.